src/utils/checkAnswer.js

```diff
@@ -0,0 +1,10 @@
+import { quizData } from '../data.js';
+
+export const checkAnswer = (questionNum) => {
+  const answer = localStorage.getItem(questionNum);
+  const correctAnswer = quizData.questions[questionNum - 1].correct;
```

Comment:
`questionNum - 1` to be questionId

Reply:
if we use question id on this then we have loop through all the question to find the match and then get the correct answer from matched id. I think this is more fast and convenient way.